templates/project-ts/src/interact.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * This script can be used to interact with the Add contract, after deploying it.
+ *
+ * We call the update() method on the contract, create a proof and send it to the chain.
+ * The endpoint that we interact with is read from your config.json.
+ *
+ * This simulates a user interacting with the zkApp from a browser, except that here, sending the transaction happens
+ * from the script and we're using your pre-funded zkApp account to pay the transaction fee. In a real web app, the user's wallet
+ * would send the transaction and pay the fee.
+ *
+ * To run locally:
+ * Build the project: `$ npm run build`
+ * Run with node:     `$ node build/src/interact.js <network>`.
+ */
+import { Mina, PrivateKey, shutdown } from 'snarkyjs';
+import fs from 'fs/promises';
+import { Add } from './Add.js';
+
+// check command line arg
+let network = process.argv[2];
+if (!network)
+  throw Error(`Missing <network> argument.
+
+Usage:
+node build/src/interact.js <network>
+
+Example:
+node build/src/interact.js berkeley
+`);
+Error.stackTraceLimit = 1000;
+
+// parse config and private key from file
+type Config = { networks: Record<string, { url: string; keyPath: string }> };
+let configJson: Config = JSON.parse(await fs.readFile('config.json', 'utf8'));
+let config = configJson.networks[network];
+let key: { privateKey: string } = JSON.parse(
+  await fs.readFile(config.keyPath, 'utf8')
+);
+let zkAppKey = PrivateKey.fromBase58(key.privateKey);
+
+// set up Mina instance and contract we interact with
+const Berkeley = Mina.BerkeleyQANet(config.url);
```

Comment:
Similar to the previous comment. Do we need this to allow interaction with other networks?